Makefile

```diff
@@ -14,6 +14,7 @@ get-deps-%:
 get-deps-pip:
 	pip install torch==1.9.0
 	pip install -r requirements.txt
+	pre-commit init
```

Comment:
Im not very familiar with makefile. I believe these commands are executed in serial, right?

Reply:
We shouldn't do this in the dependency resolution step. You'll want to add a new step that operates on the pre-commit config file so that it only runs when that file changes and add that to the main process.